rest_framework/throttling.py

```diff
@@ -35,7 +35,7 @@ def get_ident(self, request):
             client_addr = addrs[-min(num_proxies, len(xff))]
```

Comment:
As mentioned in https://github.com/tomchristie/django-rest-framework/issues/2400#issuecomment-69526373, this should be `len(addrs)` instead of `len(xff)`.